improver_tests/generate_ancillaries/test_GenerateTimezoneMask.py

```diff
@@ -393,5 +392,3 @@ def test_process(request, grid_fixture, time, grouping):
         assert (
             result.coord("UTC_offset").bounds.dtype == TIME_COORDS["UTC_offset"].dtype
         )
-
-    assert_array_equal(result.data[index][::4], expected["data"])
```

Comment:
Is it useful for this to fail when the time zone database has changed? Probably not. However, removing these lines leaves tramp data in `EXPECTED` that should be tidied up. We need some mechanism for updating our generated timezone ancillaries.